examples/plugin_example_features.py

```diff
@@ -0,0 +1,20 @@
+
+def compute_some_new_features(rtdc_ds):
+    """The function that does the heavy-lifting"""
+    circ_per_area = rtdc_ds["circ"] / rtdc_ds["area_um"]
+    bright_per_area = rtdc_ds["bright_avg"] / rtdc_ds["area_um"]
+    # returns a dictionary-like object
+    return {"circ_per_area": circ_per_area, "bright_per_area": bright_per_area}
+
+
+info = {
+    "method": compute_some_new_features,
+    "description": "This plugin will compute some possibly useless features",
+    "feature names": ["circ_per_area", "bright_per_area"],
+    "feature labels": ["", ""],
```

Comment:
`feature labels` should never be empty (Please add a check for that during instantiation). It can be omitted though (in which case it should default to `feature names`).

Reply:
`feature_labels` are added to the `dfn` file, right? Like for temporary features?

Reply:
No, we have to do it in a similar way as for the temporary features. Ancillary features don't need that, because they are only defined for features that are in the `dfn` file.

Reply:
I think it would make sense to have a shared code base for registering and deregistering plugin and temporary features in the `dfn` file.